core/src/main/java/ai/timefold/solver/core/impl/domain/valuerange/buildin/collection/ListValueRange.java

```diff
@@ -18,17 +16,15 @@ public final class ListValueRange<T> extends AbstractCountableValueRange<T> {
     private final List<T> list;
     // Built-in immutable type, like String, Integer, etc.
     private final boolean immutable;
-    private final Function<T, Integer> extractIdFunction;
     private ValueRangeCacheStrategy<T> cacheStrategy;
 
     public ListValueRange(List<T> list) {
-        this(list, false, null);
+        this(list, false);
     }
 
-    public ListValueRange(List<T> list, boolean immutable, Function<T, Integer> extractIdFunction) {
+    public ListValueRange(List<T> list, boolean immutable) {
```

Comment:
Wouldn't it be safer for the range if the immutability check was _inside_ of the class? That way, no code can make the wrong change.

Reply:
The strategy based on `BitSet` is no longer available.